### ISSUE TITLE
"Demo" preset is broken

### ISSUE THREAD

**Issue by Dumluregn** [ckeditor/ckeditor4-presets#58]
Extracted from https://github.com/ckeditor/ckeditor4-presets/pull/56#pullrequestreview-566288768.

### tl;dr:
If you build `demo` preset (only available on the `demo` branch), which is used on our webpage, and try to open sample, you get an error:

![image](https://user-images.githubusercontent.com/10883184/104327212-cf2f5c00-54ea-11eb-90e5-cf94a45d3c61.png)

Preset build flow is a bit different for the websites, but still `./build.sh demo` should produce working editor.

### Details:
https://github.com/ckeditor/ckeditor4-presets/pull/56#issuecomment-758059011
https://github.com/ckeditor/ckeditor4-presets/pull/56#issuecomment-758636864
https://github.com/ckeditor/ckeditor4-presets/pull/56#issuecomment-758644318
